apps/vault-apy/src/APYSimulator.ts

```diff
@@ -288,12 +294,12 @@ export default class APYSimulator {
         ).name(),
       }),
       network: this.#network,
-      date: new Date(prevBlock.timestamp * 1000).toISOString(),
+      date: new Date(originalTimestamp * 1000).toISOString(),
       ///////////////////////////////////////////////////////////////////////
 
       swapFees: poolFeesInPrimary.toString(),
-      blockNumber: prevBlock.number,
-      timestamp: floorToMidnight(prevBlock.timestamp),
+      blockNumber: forkBlock,
+      timestamp: floorToMidnight(originalTimestamp),
```

Comment:
@jeffywu this would save the unadjusted timestamp but the adjusted block, so they wouldn't be in sync. Could use the original block as well, but not sure if that's better